src/scss/App.scss

```diff
@@ -116,3 +116,7 @@ a {
     min-height: 100%;
   }
 }
+
+#main-content {
+  scroll-margin-top: 3rem;
```

Comment:
Can you add a comment noting where the value comes from? If we change the header height or other aspects of the layout future it could save some searching and avoid weirdness.

Reply:
comment added